lib/compile-uri/compile-params.js

```diff
@@ -1,22 +1,51 @@
-module.exports = function compileParams(hrefVariablesElement) {
-  const params = {};
+function getRequired(memberElement) {
+  const typeAttributes = memberElement.attributes.getValue('typeAttributes') || [];
```

Comment:
I wonder, if it makes sense for the minim `getValue` API To allow you to provide a "default" value. This is similiar to Python's `get(prop, default)` method.

```js
getValue('typeAttributes', []);
```

@honzajavorek As a Pythonist what do you think?

Reply:
Seems like a good idea, especially for arrays. Not sure about values such as `null`, `0`, `undefined`, and this kind of hell.